Formula/certbot.rb

```diff
@@ -154,36 +156,16 @@ class Certbot < Formula
   end
 
   def install
-    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
-    resources.each do |r|
-      r.stage do
-        system "python", *Language::Python.setup_install_args(libexec/"vendor")
-      end
-    end
-
-    # Namespace packages and .pth files aren't processed from PYTHONPATH.
-    touch libexec/"vendor/lib/python2.7/site-packages/zope/__init__.py"
-    touch libexec/"vendor/lib/python2.7/site-packages/ndg/__init__.py"
-
-    cd "acme" do
-      system "python", *Language::Python.setup_install_args(libexec)
-    end
+    virtualenv_install_with_resources
 
-    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
-    system "python", *Language::Python.setup_install_args(libexec)
-
-    cd "certbot-apache" do
-      system "python", *Language::Python.setup_install_args(libexec)
-    end
-
-    cd "certbot-nginx" do
-      system "python", *Language::Python.setup_install_args(libexec)
+    # Shipped with certbot, not external resources.
+    %w[acme certbot-apache certbot-nginx].each do |r|
+      cd r do
+        system "python", *Language::Python.setup_install_args(libexec)
+      end
```

Comment:
They aren't executables. They're modules that lump extra features into the `certbot` client. Essentially resources but vendored in the tarball rather than externally.